test/CrateTest/PDO/PDOPwAuthTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace CrateTest\PDO;
+
+use Crate\PDO\PDO;
+
+/**
+ * Tests for {@see \Crate\PDO\PDO}
+ *
+ * @coversDefaultClass \Crate\PDO\PDO
+ * @covers ::<public>
+ *
+ * @group unit
```

Comment:
don't know what this is about :)
but do we really need that header here? do we have that somewhere else?

Reply:
the header was taken over by the other PDOTest, i don't think it is needed here